docs/en/commands/context/get-context.md

```diff
@@ -1,3 +1,5 @@
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands/context/get-context.yml)
```

Comment:
Should the path not be `commands-yml/commands/context/get-context.yml`?

Reply:
Yep. Made a fix for it